### PR TITLE
OTJ 13 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/eriette_the_beguiler.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Eriette, the Beguiler
+ManaCost:1 W U B
+Types:Legendary Creature Human Warlock
+PT:4/4
+K:Lifelink 
+T:Mode$ Attached | ValidSource$ Aura | ValidTarget$ Permanent.nonLand+OppCtrl+cmcLEX | TriggerZones$ Battlefield | Execute$ TrigGainControl | TriggerDescription$ Whenever an Aura you control becomes attached to a nonland permanent an opponent controls with mana value less than or equal to that Aura's mana value, gain control of that permanent for as long as that Aura is attached to it.
+SVar:TrigGainControl:DB$ GainControl | NewController$ You | Defined$ Targeted | Duration$ UntilUnattached
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/eriette_the_beguiler.txt:7]: This is not working atm. don't know how to make it work without engine changes

File: forge-gui/res/cardsfolder/upcoming/emergent_haunting.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Emergent Haunting
+ManaCost:1 U
+Types:Enchantment
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | SVarCompare$ EQ0 | IsPresent$ Creature.Self | PresentCompare$ EQ0 | Execute$ TrigAnimate | TriggerDescription$ At the beginning of your end step, if you haven't cast a spell from your hand this turn and CARDNAME isn't a creature, it becomes a 3/3 Spirit creature with flying in addition to its other types.
+SVar:TrigAnimate:DB$ Animate | Defined$ Self | Power$ 3 | Toughness$ 3 | Types$ Creature,Spirit | Keywords$ Flying | Duration$ Permanent
+SVar:X:Count$ThisTurnCast_Card.YouCtrl+wasCastFromYourHand
+A:AB$ Surveil | Cost$ 2 U | Amount$ 2 | SpellDescription$ Surveil 1. (Look at the top card of your library. You may put it into your graveyard.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/emergent_haunting.txt:7]: wrong amount

File: forge-gui/res/cardsfolder/upcoming/eriettes_lullaby.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Eriette's Lullaby
+ManaCost:1 W
+Types:Sorcery
+A:SP$ Destroy | ValidTgts$ Creature.tapped | TgtPrompt$ Select target tapped creature | SubAbility$ DBGainLife | SpellDescription$ Destroy target tapped creature.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ 2 | SpellDescription$ You gain 2 life
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/eriettes_lullaby.txt:5]: .

File: forge-gui/res/cardsfolder/upcoming/failed_fording.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Name:Failed Fording
+ManaCost:1 U
+Types:Instant
+A:SP$ ChangeZone | ValidTgts$ Permanent.nonLand+YouDontCtrl | TgtPrompt$ Select target nonland permanent you don't control | Origin$ Battlefield | Destination$ Hand | SubAbility$ DBSurveil | SpellDescription$ Return target nonland permanent to its owner's hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/failed_fording.txt:4]: wrong tgts

File: forge-gui/res/cardsfolder/upcoming/final_showdown.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Final Showdown
+ManaCost:W
+Types:Instant
+K:Spree 
+A:SP$ Charm | Choices$ DBDebuffAll,DBChooseCard,DBDestroyAll | MinCharmNum$ 1 | CharmNum$ 3 | Spree$ True
+SVar:DBDebuffAll:DB$ AnimateAll | SpreeCost$ 1 | ValidCards$ Creature | RemoveAllAbilities$ True | SpellDescription$ All creatures lose all abilities until end of turn.
+SVar:DBChooseCard:DB$ ChooseCard | SpreeCost$ 1 | Choices$ Creature.YouCtrl | ChoiceTitle$ Choose a creature you control | SubAbility$ DBPump | SpellDescription$ Choose a creature you control. It gains indestructible until end of turn.
+SVar:DBPump:DB$ Pump | Defined$ ChosenCard | KW$ Indestructible | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/final_showdown.txt:9]: cleanup unneeded

File: forge-gui/res/cardsfolder/upcoming/fleeting_reflection.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+Name:Fleeting Reflection
+ManaCost:1 U 
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | KW$ Hexproof | SubAbility$ DBUntap | SpellDescription$ Target creature you control gains hexproof until end of turn.
+SVar:DBUntap:DB$ Untap | Defined$ Targeted | SubAbility$ DBClone | SpellDescription$ Untap that creature.
+SVar:DBClone:DB$ Clone | CloneTarget$ ParentTarget | ValidTgts$ Creature | TgtPrompt$ Select up to one target creature to copy | TargetUnique$ True | Duration$ UntilEndOfTurn | SpellDescription$ Until end of turn, it becomes a copy of up to one other target creature.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fleeting_reflection.txt:6]: "up to"

File: forge-gui/res/cardsfolder/upcoming/failed_fording.txt [hunk 7]
```diff
@@ -0,0 +1,7 @@
+Name:Failed Fording
+ManaCost:1 U
+Types:Instant
+A:SP$ ChangeZone | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent | Origin$ Battlefield | Destination$ Hand | SubAbility$ DBSurveil | SpellDescription$ Return target nonland permanent to its owner's hand.
+SVar:DBSurveil:DB$ Surveil | Amount$ 1 | ConditionPresent$ Desert | ConditionCompare$ GE1 | SpellDescription$ If you control a Desert, surveil 1. (Look at the top card of your library. You may put it into your graveyard.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/failed_fording.txt:5]: fix condition

File: forge-gui/res/cardsfolder/upcoming/eriette_the_beguiler.txt (hunk 1 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/eriette_the_beguiler.txt]: Thanks seem to be working now 
image

**Pull request merged**